mne/externals/__init__.py

```diff
@@ -4,3 +4,4 @@
 from . import jdcal
 from . import six
 from . import tempita
+from . import pymatreader
```

Comment:
Don't do this import, it will force a hard `h5py` dependency. Instead later you should be able to do `from ..externals.pymatreader import read_mat` and it should still work